app/_locales/en/messages.json

```diff
@@ -1726,6 +1732,9 @@
   "viewAccount": {
     "message": "View Account"
   },
+  "viewOnCustomBlockExplorer": {
+    "message": "View at $1"
```

Comment:
View *on* (to be consistent with "View on Etherscan" below

Reply:
Hmm... I don't know what the correct grammar is here. `$1` will always be a url. "View at http://example.com" sounds a bit more correct... but I don't know why and I might be wrong.

Reply:
I am not the best candidate to comment on English grammar 😥 

I am sure your intuition is more correct than mine.